api/docker-compose.yaml

```diff
@@ -36,10 +36,10 @@ services:
       - elastic
     build:
       context: ./src/dev/server
-    user: ${UID}
+    user: ${USER_ID}
```

Comment:
Is there a way to reuse some of this config info so you don't have to re-specify it each time?

Reply:
Maybe it could have, but repetition in configuration isn't too detrimental (and can sometimes be beneficial for operations, requiring less head-scratching to understand the state of things).

Anyway, this may be a moot point as I think I found an alternative which didn't require any changes here. (See my longer reply to CH's comment.)